game-reporter/src/queue.rs

```diff
@@ -263,6 +263,7 @@ fn process_reports(queue: &GameReporterQueue, event: ProcessingEvent) {
 }
 
 /// The true inner error, minus any metadata.
+#[allow(dead_code)]
```

Comment:
This is probably fine, but it might be more explicit to tag it on each enum variant kind - and maybe we should add a doc comment noting why we did it?

i.e, `the compiler thinks these are unused, but they're not - debug impls will render them over the Dolphin logging interface and the compiler just doesn't see that` or something?